activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -575,9 +573,14 @@ def build_insert_sql(insert) # :nodoc:
         "INSERT #{insert.into} #{insert.values_list}"
       end
 
+      def check_version!
+      end
```

Comment:
I think it should still be `check_version`. Our style is to only use a bang method when there's a corresponding non-bang method. Additionally this method isn't dangerous so I think `check_version` is more correct over `check_version!`. Thoughts?

Reply:
Yeah, that makes sense. I made it `check_version!` because it could raise an exception, but it's better to stick with conventions.